packages/dm-core-plugins/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@development-framework/dm-core-plugins",
+  "license": "MIT",
+  "version": "1.0.1",
+  "main": "dist/index.js",
+  "dependencies": {
+    "@equinor/eds-core-react": "^0.29.1",
+    "@equinor/eds-icons": "^0.18.0",
+    "@equinor/eds-tokens": "^0.9.0",
+    "axios": "^0.27.2",
+    "golden-layout": "1.5.9",
+    "highlight.js": "^10.7.2",
+    "js-yaml": "^4.1.0",
+    "mermaid": "^10.0.0",
+    "react-contextmenu": "^2.11.0",
+    "react-copy-to-clipboard": "^5.1.0",
+    "react-notifications": "^1.4.3",
+    "styled-components": "^5.1.0",
+    "typescript": "^4.5.5",
+    "object-path": "^0.11.8",
+    "react-hook-form": "^7.31.2"
+  },
+  "devDependencies": {
+    "@development-framework/dm-core": ">=1.0.37",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/react-hooks": "^7.0.2",
+    "@testing-library/user-event": "14.4.3",
+    "@types/highlightjs": "^9.12.2",
+    "@types/jest": "^27.4.0",
+    "@types/js-yaml": "^4.0.5",
+    "@types/react": "17.0.39",
+    "@types/react-dom": "17.0.11",
+    "@types/react-router-dom": "^5.3.3",
+    "@types/styled-components": "^4.1.18",
+    "jest": "^27.1.3",
+    "jest-environment-jsdom": "^27.1.1",
+    "react-router-dom": ">=5.1.2",
+    "react-test-renderer": "17.0.2",
+    "ts-jest": "^27.1.3",
+    "ts-loader": "^9.3.1",
+    "typescript": "^4.5.5",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "styled-components": ">=5.1.0"
```

Comment:
this is added to all three deps list. I suggest having it in the normal deps list only

Reply:
update: added styled-components to peer dependency, as recommended by Stig, but I had to add styled-components to dev dependencies also to make tests run.
Removed from normal dependencies list.